src/Features/CSharp/Portable/UseDefaultLiteral/CSharpUseDefaultLiteralCodeFixProvider.cs

```diff
@@ -60,8 +66,9 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             foreach (var originalDefaultExpression in originalNodes)
             {
                 var defaultExpression = currentRoot.GetCurrentNode(originalDefaultExpression);
+                var verifySemantics = nodesToVerify.Contains(originalDefaultExpression);
```

Comment:
it would be exactly the same, bu ti would find it easier to read this as "var skipVerification = !..." and then "if (skipVerification || ...)"

There's something about "!a || b" that i find taxing :)